packages/erc3k/contracts/IERC3000.sol

```diff
@@ -8,20 +8,25 @@ pragma experimental ABIEncoderV2;
 import "./ERC3000Data.sol";
 import "./IERC3000Executor.sol";
 
+// Note: why not defining functions as external?
```

Comment:
See note in #42. The ABI Decoder makes the compiler crash when getting structs from calldata. Therefore the functions cannot be external and we have to make public so the struct is in memory